.gitignore

```diff
@@ -1,2 +1,4 @@
 target/
 **/*.rs.bk
+src/asset.in
+frontend/dist/
```

Comment:
Maybe for now we have the dependency then make it work in a more pluggable way for when people use it? I think this is what you wanted to do eventually correct?